db/migrate/20161228201733_add_full_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddFullNameToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :first_name, :string
```

Comment:
what about factory changes? 
we need to have validation for first_name, last_name and generate them in factory girl with faker